module/VuFind/src/VuFind/I18n/TranslatableString.php

```diff
@@ -50,10 +50,12 @@ class TranslatableString implements TranslatableStringInterface
      *
      * @var string
      */
-    protected $displayString;
+    protected $displayString = '';
```

Comment:
I didn't think it was necessary to initialize a property that is initialized by the constructor. Did adding this specifically change something?

Reply:
Oh, actually I must've been wrong with this one. Was doing a second task at the same time and mixed them up.

Reply:
If you don't mind reverting this unnecessary change, we can at least merge the rest to add the missing annotation!

Reply:
Will convert this to draft and find the real culprit.

Reply:
Seems like someone is actually putting a null variable into the translate function and it starts to fill the php_error.log with dep notices in our end. Will need to find the culprit there. Sorry for the confusion!